ntbs-service/Models/Entities/Alert.cs

```diff
@@ -59,6 +60,21 @@ public bool CaseManagerAllowedForTbService
                 return CaseManager.CaseManagerTbServices.Any(c => c.TbServiceCode == TbServiceCode);
             }
         }
+
+        [NotMapped]
+        public string NotificationTbServiceCode { get; set; }
+        public bool TransferDestinationNotCurrentTbService
+        {
+            get
+            {
+                if (TbServiceCode == NotificationTbServiceCode && AlertType == AlertType.TransferRequest)
+                {
+                    return false;
+                }
+
+                return true;
+            }
+        }
```

Comment:
This can be expression bodied:
i.e.
```
TransferDestinationNotCurrentTbService => AlertType != AlertType.TransferRequest || TbServideCode != NotificationTbServiceCode
```